lib/knife/changelog/policyfile.rb

```diff
@@ -22,7 +22,7 @@ def all_cookbooks
 
       # return true if cookbook is not already listed as dependency
       def new_cookbook?(name)
-        policy.send(:best_source_for, name).nil?
+        lock.solution_dependencies.cookbook_dependencies.keys.find { |dep| dep.name == name }.nil?
```

Comment:
For me it should belong to another review.

Reply:
Sure.

Reply:
Cool, thanks.